docs/source/discovery-overview.rst

```diff
@@ -30,6 +30,11 @@ The **discovery service** improves this process by having the peers compute
 the needed information dynamically and present it to the SDK in a consumable
 manner.
 
+Starting with v2.4, the :doc:`gateway` interacts with service discovery in order to determine
+which peers are required to endorse a transaction, and which orderer nodes to send
+the transaction.  Consequently, client applications written using the new gateway
+SDKs do not need to interact with service discovery at all.
```

Comment:
"do not need to" implies that they still can. Can they? 
... alternatives are "do not interact" or "should not interact" ...

Reply:
changed to "do not interact directly with..."